plugins/dependencies/src/main/resources/removed-dependencies-versions-keys.txt

```diff
@@ -107,3 +107,4 @@ androidx.health..health-connect-client=version.androidx.health-connect-client
 com.google.accompanist..accompanist-glide=version.google.accompanist
 com.google.accompanist..accompanist-imageloading-core=version.google.accompanist
 app.cash.molecule..molecule-testing=version.app.cash.molecule
+androidx.compose.runtime..runtime-tracing=version.androidx.compose.runtime
```

Comment:
what's the reason for this commit?

Reply:
https://github.com/Splitties/refreshVersions/actions/runs/5467565767 vs. https://github.com/Splitties/refreshVersions/actions/runs/5469518362

Reply:
I see thanks!